gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -81,7 +83,10 @@
   public GobblinApplicationMaster(String applicationName, ContainerId containerId, Config config,
       YarnConfiguration yarnConfiguration) throws Exception {
     super(applicationName, containerId.getApplicationAttemptId().getApplicationId().toString(),
-        GobblinClusterUtils.addDynamicConfig(config), Optional.<Path>absent());
+        GobblinClusterUtils.addDynamicConfig(config)
+        .withFallback(ConfigFactory.parseMap(
+            ImmutableMap.of(GobblinClusterConfigurationKeys.CLEAN_UP_JOBS_ON_MANAGER_START, "true"))),
```

Comment:
Should it be ImmutableMap.of(GobblinClusterConfigurationKeys.CLEAN_UP_JOBS_ON_MANAGER_START, DEFAULT_CLEAN_UP_JOBS_ON_MANAGER_START) ?

Reply:
No, this is explicitly setting it to "true" for the Gobblin app master. All other gobblin cluster modes use the default of "false".